### DISCUSSION TITLE
Show a tag if the clipboard contains a youtube link

### DISCUSSION THREAD

**Discussion by e5pe** [SilentVoid13/Templater#146] [Help] [answered]
Hi, I love this plugin. Thank you so much for your work!

I tried to make a template, so when I copy some link to a youtube video it returns a youtube tag, like this `#youtube`, but I can't make it work. Could you help me?

This is the code:
```js
<%*
var tag =  "#link";
const clipboard =tp.system.clipboard();

 if (clipboard.contains("youtube") || clipboard.contains("youtu.be")) {
     tag =  "#youtube";
} %>
<%* tR += tag %>

```

Thank you 😃

**Comment by SilentVoid13** [comment, marked as answer]
Hey @e5pe, you're missing the `await` keyword, because `tp.system.clipboard` is an asynchronous function.
The correct template would be:

```js
<%*
var tag =  "#link";
const clipboard = await tp.system.clipboard();
if (clipboard.contains("youtube") || clipboard.contains("youtu.be")) {
    tag =  "#youtube";
} 
tR += tag;
%>
```

**Reply by e5pe** [reply]
Thank you!
